package.json

```diff
@@ -12,15 +15,18 @@
     "@types/react-dom": "^18.2.14",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
+    "react-router-dom": "^6.21.0",
     "react-scripts": "5.0.1",
+    "styled-reset": "^4.5.1",
```

Comment:
Would you explain what is this package about?

Reply:
styled-reset is for reset default browser style like normalize.css

Reply:
I did research about what it does. Please provide more context to it next time.

Reply:
Okay i got it!